src/codeu/chat/common/Interests.java

```diff
@@ -0,0 +1,52 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collection;
+
+import codeu.chat.util.Serializer;
+import codeu.chat.util.Serializers;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public final class Interests {
+
+  public static final Serializer<Interests> SERIALIZER = new Serializer<Interests>() {
+
+    @Override
+    public void write(OutputStream out, Interests value) throws IOException {
+
+      Serializers.collection(Uuid.SERIALIZER).write(out, value.interests);
+      Time.SERIALIZER.write(out, value.lastStatusUpdate);
+      Time.SERIALIZER.write(out, value.creation);
+
+    }
+
+    @Override
+    public Interests read(InputStream in) throws IOException {
+
+      return new Interests(
+          Serializers.collection(Uuid.SERIALIZER).read(in),
+          Uuid.SERIALIZER.read(in),
+          Time.SERIALIZER.read(in),
+          Time.SERIALIZER.read(in)
+      );
+
+    }
+  };
+
+  public final Collection<Uuid> interests;
+  public final Uuid id;
```

Comment:
Didn't we agree in the design doc that interests didn't need their own UUIDs and could just be looked up in the server via their corresponding users' UUIDs? If so, then how is this used here?